src/advanced-settings/AdvancedSettings.jsx

```diff
@@ -5,28 +5,47 @@ import { Container, Button, Layout } from '@edx/paragon';
 import { CheckCircle, Info, WarningFilled } from '@edx/paragon/icons';
 import { FormattedMessage, injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import { fetchCourseAppSettings, updateCourseAppSetting, fetchProctoringExamErrors } from './data/thunks';
-import { getCourseAppSettings, getSavingStatus, getProctoringExamErrors } from './data/selectors';
+import {
+  getCourseAppSettings, getSavingStatus, getProctoringExamErrors, getSendRequestErrors,
+} from './data/selectors';
 import SettingCard from './setting-card/SettingCard';
 import SettingAlert from './setting-alert/SettingAlert';
 import SettingsSidebar from './settings-sidebar/SettingsSidebar';
 import { RequestStatus } from '../data/constants';
-import { parseArrayOrObjectValues } from '../utils';
+import { parseArrayOrObjectValues, removeExtraQuotes, validateAdvancedSettingsData } from '../utils';
 import messages from './messages';
 import AlertProctoringError from '../generic/AlertProctoringError';
+import ModalError from './modal-error/ModalError';
 
 const AdvancedSettings = ({ intl, courseId }) => {
   const advancedSettingsData = useSelector(getCourseAppSettings);
   const savingStatus = useSelector(getSavingStatus);
   const proctoringExamErrors = useSelector(getProctoringExamErrors);
+  const settingsWithSendErrors = useSelector(getSendRequestErrors) || {};
   const dispatch = useDispatch();
   const [saveSettingsPrompt, showSaveSettingsPrompt] = useState(false);
   const [showDeprecated, setShowDeprecated] = useState(false);
+  const [errorModal, showErrorModal] = useState(false);
   const [editedSettings, setEditedSettings] = useState({});
+  const [errorFields, setErrorFields] = useState([]);
+  const [successAlert, showSuccessAlert] = useState(false);
```

Comment:
boolean states should not be named like this, please rename to `[showSuccessAlert, setShowSuccessAlert]`. same with other states added in this PR